sdk/js/src/api/http.js

```diff
@@ -59,11 +58,13 @@ class Http {
     }
   }
 
-  async handleRequest (method, endpoint, rawPayload) {
-    const payload = rawPayload ? Marshaler.payload(rawPayload) : {}
+  async handleRequest (method, endpoint, payload = {}, config) {
     const component = this._parseStackComponent(endpoint)
     try {
-      return await this[component][method](endpoint, payload)
+      if (method === 'get') {
+        return await this[component][method](endpoint, config)
```

Comment:
why not passing an empty payload here? 
why do we need that separation?

Reply:
To me, it seemed weird to define a get function with a payload argument that is always discarded.

Reply:
Disregard the above, I actually mixed that up. You are right indeed. I wrongfully thought, the [different signature](https://github.com/axios/axios#request-method-aliases) of the get method would make that necessary.